<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="startSearch">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始搜索</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="musicName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>歌曲名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="musicName_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索结果</string>
   </property>
  </widget>
  <widget class="QListView" name="musicList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>551</width>
     <height>581</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="resultNum">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="musicName_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>下载编号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startDownload">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始下载</string>
   </property>
  </widget>
  <widget class="QLabel" name="musicName_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>下载路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="chooseDirectoryPath">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseDirectory">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
